similarity-data-impl/src/main/java/org/phenotips/data/similarity/script/ExomiserViewScriptService.java

```diff
@@ -63,91 +60,61 @@
     @Named("edit")
     private AccessLevel editAccess;
 
+    /** Manager to allow access to patient exome data. */
+    @Inject
+    @Named("exomiser")
+    private ExomeManager exomeManager;
+
     /**
-     * Gets the top K genes by score from a patient.
+     * Checks if a patient has a valid Exomiser genotype.
      *
-     * @param patient A valid patient
+     * @param patient A valid {@link #Patient}
```

Comment:
The description for `@param`, `@return` and `@throws`, in case it is only one sentence, should start with lowercase and not end with a dot. It's not your fault here, though.